json_annotation/lib/src/json_serializable.dart

```diff
@@ -64,6 +65,13 @@ class JsonSerializable {
   /// This setting has no effect if [createFactory] is `false`.
   final String? constructor;
 
+  /// If `true` (defaults to false), a private, static `_$ExampleJsonMeta`
+  /// constant is created in the generated part file.
+  ///
+  /// This constant can be used by other code-generators to support features
+  /// such as [fieldRename].
+  final bool? createJsonMeta;
```

Comment:
"meta" is pretty vague here. Maybe `createFieldMap`? Let's talk before you start rewriting everything...

Reply:
xFieldMap sounds good to me